### DISCUSSION TITLE
Do changes to plotly.js propagate here?

### DISCUSSION THREAD

**Discussion by andresrcs** [plotly/plotly.R#2229] [Q&A]
I've made a PR for correcting the `es-PE` locale on `plotly.js` . Will these changes propagate here when merged or do I need to open a new PR here to correct this on `plotly.R`?

https://github.com/plotly/plotly.js/pull/6451

**Comment by TucoFernandes** [comment]
I was wondering the same. You can see a lot of new functions and things happening on plotly.js and plotly for python. But R plotly looks the same in the last 2 years.

**Comment by cpsievert** [comment]
The version of plotly.js that is bundled with the R package can be updated by running this script https://github.com/plotly/plotly.R/blob/master/tools/update_plotlyjs.R

I'm typically pretty conservative about updating plotly.js; but that said, it has been almost a year since the last update (https://github.com/plotly/plotly.R/pull/2096), so I'd be open to a PR that does a version update.
